_data/toc.yaml

```diff
@@ -1323,10 +1323,6 @@ manuals:
           section:
           - title: Configure your deployment
             path: /ee/ucp/interlock/config/
-          - title: Using a custom extension template
```

Comment:
@paigehargrave @ehazlett Is the decision to add information on custom templates to a KB article?